actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -2,6 +2,25 @@
 
 module ActionDispatch
   module SystemTesting
+    # Deletes screenshots older than the provided TTL (time to live) in seconds.
+    #
+    # This method is supposed to be called in the beginning of a test suite run and after
+    # test files and configuration have been loaded.
+    def self.rotate_screenshots(ttl:)
```

Comment:
`ttl` is probably to low-level; other options are `max_age: Number` or `older_than: Time`.

Reply:
Here are my suggestions:
```
ActionDispatch::SystemTesting.delete_screenshots(older_than: 2.days)
ActionDispatch::SystemTesting.keep_screenshots(for: 2.days)
```